#! /bin/sh /usr/share/dpatch/dpatch-run
## 39_scripts__mysqld_safe.sh__port_dir.dpatch by <adconrad@0c3.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Make sure permissions on /var/run/mysqld are correct on a tmpfs.

@DPATCH@

--- old/scripts/mysqld_safe.sh	2005-12-22 06:40:02.000000000 +1100
+++ new/scripts/mysqld_safe.sh	2006-02-24 17:32:08.000000000 +1100
@@ -211,6 +211,7 @@
 then
   mkdir $mysql_unix_port_dir
   chown $user $mysql_unix_port_dir
+  chmod 755 $mysql_unix_port_dir
 fi
 
 # Use the mysqld-max binary by default if the user doesn't specify a binary
